src/static/js/donations.js

```diff
@@ -488,6 +502,59 @@ $(function() {
                 return false;
             });
 
+            $("#button-cardcom-token").click(function() {
+                payment_processor_confirmation_dialog("cardcom");
+                return false;
+            });
+
+            const payment_processor_confirmation_dialog= async function(processor_name) {
+                let row = tableform.table_selected_row(donations.table);
```

Comment:
function should at least have cardcom in the name. Might be nice to move it out at least to a higher level as a member of donations object or if there is going to be more cardcom UI stuff, to a separate file?

Reply:
I put it in the same level as the other buttons' functions and at first I thought this (just like the payment_processor_popup function) can actually be implemented with other payment providers, but you're right that the extra ID keys do contain Cardcom in their names... 

Perhaps we can just add a per-provider function to get the confirmation text (or just a switch/case if block by processor_name)?